presto-spi/src/main/java/io/prestosql/spi/StandardErrorCode.java

```diff
@@ -64,6 +64,47 @@
     INVALID_SPATIAL_PARTITIONING(0x0000_0029, USER_ERROR),
     INVALID_ANALYZE_PROPERTY(0x0000_002A, USER_ERROR),
     TYPE_NOT_FOUND(0x0000_002B, USER_ERROR),
+    CATALOG_NOT_FOUND(0x0000_002C, USER_ERROR),
+    SCHEMA_NOT_FOUND(0x0000_002D, USER_ERROR),
+    TABLE_NOT_FOUND(0x0000_002E, USER_ERROR),
+    COLUMN_NOT_FOUND(0x0000_002F, USER_ERROR),
+    ROLE_NOT_FOUND(0x0000_0030, USER_ERROR),
+    SCHEMA_ALREADY_EXISTS(0x0000_0031, USER_ERROR),
+    TABLE_ALREADY_EXISTS(0x0000_0032, USER_ERROR),
+    COLUMN_ALREADY_EXISTS(0x0000_0033, USER_ERROR),
+    ROLE_ALREADY_EXISTS(0x0000_0034, USER_ERROR),
+    NAMED_QUERY_ALREADY_EXISTS(0x0000_0035, USER_ERROR),
+    MISSING_COLUMN_NAME(0x0000_0036, USER_ERROR),
+    MISSING_CATALOG_NAME(0x0000_0037, USER_ERROR),
+    MISSING_SCHEMA_NAME(0x0000_0038, USER_ERROR),
+    INVALID_SCHEMA_NAME(0x0000_0039, USER_ERROR),
```

Comment:
This is only used in one place, `MetadataUtil`, as a helper method for the `QualifiedName` passed to schema statements like `CREATE SCHEMA`. This can probably use `SYNTAX_ERROR`, like the related `Too many dots in table name` error.